src/cadence/testnet/getNFTs.cdc

```diff
@@ -2360,3 +2361,34 @@ pub fun getOwnersNFT(owner: PublicAccount, id: UInt64): NFTData? {
         }
     )
 }
+
+// https://flow-view-source.com/testnet/account/0x161bcffdf67a19bc/contract/Metaverse
+pub fun getOzoneMetaverseNFT(owner: PublicAccount, id: UInt64): NFTData? {
+    let contract = NFTContract(
+        name: "Metaverse",
+        address: 0x161bcffdf67a19bc,
+        storage_path: "Metaverse.CollectionStoragePath",
+        public_path: "Metaverse.CollectionPublicPath",
+        public_collection_name: "Metaverse.MetaverseCollectionPublic",
+        external_domain: "https://ozonemetaverse.io"
+    )
+
+    let col = owner.getCapability(Metaverse.CollectionPublicPath)
+        .borrow<&{Metaverse.MetaverseCollectionPublic}>()
+    if col == nil { return nil }
+
+    let nft = col!.borrowMetaverse(id: id)
+    if nft == nil { return nil }
+
+    return NFTData(
+        contract: contract,
+        id: nft!.id,
+        uuid: nft!.uuid,
+        title: nil,
+        description: nil,
+        external_domain_view_url: nil,
+        token_uri: nil,
+        media: [],
```

Comment:
Is the media and metadata stored on-chain?

Reply:
Currently we don't store media or metadata on the chain. This is the initial integration of our Metaverse contract which will be refined as we finalize our metaverse implementation roadmap.